R/readWav.R

```diff
@@ -8,26 +8,36 @@ readWav = function(filename, start = 1, end = 100, units = "minutes") {
   B = as.data.frame(B, stringsAsFactors = TRUE)
   #-------------------------------------------------------
   # extract info from header: fileEncoding does not seem to be consistent, so try variants
-  header = c()
-  Nlines = 16
-  while (length(header) == 0 | length(grep("Scale-3",header)) == 0 |
+  header = header_rownames = c()
+  Nlines = 18
+  options(warn = -1)
```

Comment:
An alternative approach to `options(warn  = -1)` could be to use `suppressWarnings()` around each of the 4 `read.csv()` calls. In that way it is more clear which warnings we are trying to suppress.

Reply:
Done =)